apps/designer/app/designer/features/style-panel/sections/layout.tsx

```diff
@@ -0,0 +1,101 @@
+import { Grid } from "@webstudio-is/design-system";
+import type { RenderCategoryProps } from "../style-sections";
+import { GridControl, LockControl, ShowMore } from "../controls";
+
+const CSSLayoutSection = {
+  alignItems: "center",
+  gap: "$space$styleSection",
+  "& > [data-type=iconbuttonwithmenu]": {
```

Comment:
I am not sure why layout knows about that specific component, seems like they should be fully decoupled

Reply:
sorry, I just now understood that the layout is the name of the section, not a "layout component"